inc/utility/class-handlestransfers.php

```diff
@@ -0,0 +1,348 @@
+<?php
+/**
+ * @author  Pressbooks <code@pressbooks.com>
+ * @license GPLv3 (or any later version)
+ */
+
+namespace Pressbooks\Utility;
+
+use function Pressbooks\Image\is_valid_image;
+use function Pressbooks\Image\proper_image_extension;
+use function Pressbooks\Redirect\force_download;
+use Pressbooks\Transferable;
+
+/**
+ * This trait allows exporting taxonomies.
+ */
+trait HandlesTransfers {
+	/**
+	 * Adds the required filters to handle exporting and importing data.
+	 *
+	 * @param $obj
+	 * @return void
+	 */
+	public function bootExportable( Transferable $obj ) {
+		add_action( self::TAXONOMY . '_pre_add_form', [ $obj, 'renderImportForm' ] );
```

Comment:
@fdalcin I wonder if it would make sense to add the `TAXONOMY` constant as part of the interface, given it is used here. What do you think, it is necessary?

Reply:
I don't think it's possible to add a constant a constant as part of the interface because we won't be able to override it.

Reply:
you're right Dalcin!